src/listings/reserved_ranges.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Reserved Ranges"
+---
+
+# Actor Reserved Ranges
+
+| Actor                | ID |
+|---|---|
+| SystemActor          | 1 |
+| InitActor            | 2 |
+| CronActor            | 3 |
+| AccountActor         | 4 |
+| StoragePowerActor    | 5 |
+| StorageMinerActor    | 6 |
+| StorageMarketActor   | 7 |
+| PaymentChannelActor  | 8 |
```

Comment:
Call this "Actor ID" reserved ranges.

I think you may have confused BuiltinActorID actor-type enumeration which I have literally just deleted (#795) from `actor.id`, with the reserved actor _ID addresses_ defined in `address.go`. These are the ones we want to refer to here. Sorry for the confusion. There's also

```
const FirstNonSingletonActorId = 100
```
So the reserved range is all values lower than 100 for singleton actors.

Reply:
Fixed.